Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Repository.rst

```diff
@@ -79,6 +79,11 @@ they should be set in the method :php:`initializeObject()` method.
     Depending on the query settings, hidden or even deleted objects can become
     visible. This might cause sensitive information to be disclosed. Use with care.
 
+..  attention::
+    Since introduction of Services.yaml with TYPO3 10 you may have to flush
```

Comment:
```suggestion
    Since introduction of :ref:`dependency injection via Services.yaml <dependency-injection-in-extensions>` 
    with TYPO3 v10 you may have to flush
```
Let us link this